playbooks/openstack/configuration.md

```diff
@@ -171,17 +171,103 @@ another external DNS server.
 
 ## Kuryr Networking Configuration
 
-If you opt to use Kuryr for networking, make sure that you review all
-the Kuryr options. The following parameters should be uncommented in
-`inventory/group_vars/all.yml`:
+Kuryr is an SDN that uses OpenStack Neutron. This prevents the double overlay
+overhead one would get when running OpenShift on OpenStack using the default
+OpenShift SDN.
+
+
+### OpenStack Requirements
+
+Kuryr has a few additional requirements on the underlying OpenStack deployment:
+
+* The Trunk Ports extension must be enabled:
+  * https://docs.openstack.org/neutron/pike/admin/config-trunking.html
```

Comment:
The linked documentation doesn't mention restarting 'neutron-server'; may be worth mentioning here?